src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -296,6 +296,24 @@ internal static bool TryConvert(RazorCompletionItem razorCompletionItem, out Com
                         completionItem = parameterCompletionItem;
                         return true;
                     }
+                case RazorCompletionItemKind.Text:
+                    {
+                        var descriptionInfo = razorCompletionItem.GetDirectiveCompletionDescription();
```

Comment:
Given this is not a directive, we shouldn't reuse `GetDirectiveCompletionDescription()` for this. Instead I suggest creating your own extension method and using it.